src/components/fields/location-field/location-modal/location-search/location-search.styles.ts

```diff
@@ -42,23 +42,23 @@ export const SearchBarContainer = styled.div<{ hasScrolled?: boolean }>`
 	}
 `;
 
-export const SearchBarIconButton = styled.button`
+export const SearchBarIconButton = styled.button<{ disableCursor?: boolean }>`
 	display: flex;
 	width: fit-content;
 	align-items: center;
 	background: none;
 	border: none;
 	padding: 0;
 	margin: 0;
-	cursor: pointer;
+	cursor: ${({ disableCursor }) => (disableCursor ? `not-allowed;` : `pointer;`)};
 `;
 
 export const SearchBarIcon = styled.img`
 	width: 1rem;
 	height: auto;
 `;
 
-export const SearchBarInput = styled.input`
+export const SearchBarInput = styled.input<{ disableInput?: boolean }>`
 	border: none;
 	width: 100%;
 	margin: 0;
```

Comment:
remember to clear the grey background if field is disabled, notice this on chrome

Reply:
ah sorry didnt see the background while using the external monitor, updated.